src/impl/fsm/ConcurrentFSM.ts

```diff
@@ -24,9 +24,6 @@ export class ConcurrentFSM<F extends FSM> extends FSMImpl {
 
     public constructor(fsms: ReadonlyArray<F>) {
         super();
-        if (fsms.length < 2) {
```

Comment:
Why do you remove this condition?

Reply:
To allow for a Pan with one finger if needed